### ISSUE TITLE
[Paper Note] Extracting alignment data in open models, Federico Barbero+, arXiv'25, 2025.10

### ISSUE THREAD

**Issue by AkihikoWatanabe** [AkihikoWatanabe/paper_notes#3399]
# URL
- https://arxiv.org/abs/2510.18554
# Authors
  - Federico Barbero
  - Xiangming Gu
  - Christopher A. Choquette-Choo
  - Chawin Sitawarin
  - Matthew Jagielski
  - Itay Yona
  - Petar Veličković
  - Ilia Shumailov
  - Jamie Hayes
# Abstract
  - In this work, we show that it is possible to extract significant amounts of alignment training data from a post-trained model -- useful to steer the model to improve certain capabilities such as long-context reasoning, safety, instruction following, and maths. While the majority of related work on memorisation has focused on measuring success of training data extraction through string matching, we argue that embedding models are better suited for our specific goals. Distances measured through a high quality embedding model can identify semantic similarities between strings that a different metric such as edit distance will struggle to capture. In fact, in our investigation, approximate string matching would have severely undercounted (by a conservative estimate of $10\times$) the amount of data that can be extracted due to trivial artifacts that deflate the metric. Interestingly, we find that models readily regurgitate training data that was used in post-training phases such as SFT or RL. We show that this data can be then used to train a base model, recovering a meaningful amount of the original performance. We believe our work exposes a possibly overlooked risk towards extracting alignment data. Finally, our work opens up an interesting discussion on the downstream effects of distillation practices: since models seem to be regurgitating aspects of their training set, distillation can therefore be thought of as indirectly training on the model's original dataset.
# Translation (by gpt-4o-mini)
- 本研究では、ポストトレーニングモデルから大量のアライメントトレーニングデータを抽出することが可能であることを示します。これは、モデルの特定の能力（長文コンテキスト推論、安全性、指示の遵守、数学など）を改善するために役立ちます。関連する研究の大部分がトレーニングデータ抽出の成功を文字列マッチングを通じて測定することに焦点を当てているのに対し、私たちは埋め込みモデルが私たちの特定の目標により適していると主張します。高品質の埋め込みモデルを通じて測定された距離は、編集距離のような異なるメトリックでは捉えにくい文字列間の意味的類似性を特定できます。実際、私たちの調査では、近似文字列マッチングは、メトリックを減少させる些細なアーティファクトのために、抽出可能なデータ量を保守的に推定しても$10\times$も過小評価してしまうことがわかりました。興味深いことに、モデルはSFTやRLなどのポストトレーニングフェーズで使用されたトレーニングデータを容易に再生することがわかりました。このデータは、ベースモデルのトレーニングに使用され、元のパフォーマンスの意味のある量を回復することができます。私たちは、この研究がアライメントデータ抽出に対する見落とされがちなリスクを明らかにしていると考えています。最後に、私たちの研究は蒸留手法の下流効果に関する興味深い議論を開きます。モデルがトレーニングセットの側面を再生しているように見えるため、蒸留はモデルの元のデータセットに間接的にトレーニングしていると考えることができます。
# Summary (by gpt-4o-mini)
- 本研究では、ポストトレーニングモデルからアライメントトレーニングデータを抽出する方法を示し、埋め込みモデルが特定の能力向上に適していると主張します。文字列マッチングに依存せず、意味的類似性を捉えることで、抽出可能なデータ量を過小評価するリスクを明らかにしました。また、モデルはポストトレーニングフェーズで使用されたデータを再生でき、元のパフォーマンスを回復可能であることを示しました。研究は蒸留手法の影響についても議論します。

**Comment by AkihikoWatanabe** [comment]
元ポスト:https://x.com/fedzbar/status/1980994748124172539?s=46&t=Y6UuIHB0Lv0IpmFAjlc2-Q

**Comment by AkihikoWatanabe** [comment]
Magpieのような話だろうか？

**Comment by AkihikoWatanabe** [comment]
関連:
- #2094
